util/memory.go

```diff
@@ -0,0 +1,44 @@
+package util
+
+import (
+	"fmt"
+
+	mem "github.com/mackerelio/go-osstat/memory"
+)
+
+type Memory struct {
+	Bytes uint64
+}
+
+func (m Memory) String() string {
+	calc := float64(m.Bytes)
+	i := 0
+	for ; calc > 1024; i++ {
+		calc /= 1024
+	}
+	size := "B"
+	switch i {
+	case 1:
+		size = "KiB"
+	case 2:
+		size = "MiB"
+	case 3:
+		size = "GiB"
+	case 4:
+		size = "TiB"
+	case 5:
+		size = "PiB"
+	case 6:
+		size = "EiB"
```

Comment:
:trollface:

Reply:
Still can add ZiB and YiB, but they are bigger than 2^64